.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
For future reference - I'd add the `.idea` folder to your .gitignore - no need to store this random info about your own individual PHPStorm set up!

Don't worry about it for now 😄

Reply:
Forgot I could do that!